server/stdio.go

```
@@ -50,30 +69,27 @@ func (s *StdioServer) Listen(
 	stdout io.Writer,
 ) error {
 	// Set a static client context since stdio only has one client
-	ctx = s.server.WithContext(ctx, NotificationContext{
-		ClientID:  "stdio",
-		SessionID: "stdio",
-	})
+
+	s.server.RegisterSession(&stdioSessionInstance)
```

golangci-lint (1.62.2)

Error return value of `s.server.RegisterSession` is not checked

(errcheck)